src/chrome/local-runtime.ts

```diff
@@ -197,6 +205,36 @@ export default class LocalRuntime {
     }
   }
 
+  private async history(): Promise<NavigationHistory> {
+    const {Page} = this.client
+    await Promise.all([Page.enable()])
+    return await Page.getNavigationHistory()
+  }
+
+  private async back(): Promise<void> {
+    const history = await this.history()
+    if (history && history.currentIndex > 0) {
+      const {Page} = this.client
+      await Page.navigateToHistoryEntry({entryId: history.entries[history.currentIndex - 1].id})
+      await Page.loadEventFired()
```

Comment:
Does `loadEventFired()` get triggered when navigating back on a hash URL? E.g. From blah.html#section2 back to blah.html#section1?